Update device-checklist.md

YesCT opened the pull request:
todo: update more

YesCT commented:
I'm coming back to this.

YesCT commented:
ready for review.

deviantintegral commented on appendix/device-checklist.md:
```diff
@@ -10,33 +10,35 @@ Before starting, gather a list of all devices to audit that are used for Lullabo
-- [ ] Confirm VPN app (Proton VPN) is setup and the team member is aware of the "split tunnel" option.
-- [ ] Confirm 1Password is installed with browser extensions.
-- [ ] Confirm an adblocker (uBlock origin, AdGuard) is installed in the default browser.
+- [ ] Confirm all mobile devices have a [screen timeout set to no more than 10 minutes](https://security.lullabot.com/physical/locking.html).
```
This should include desktop computers too, so something like "all computers and mobile devices".

deviantintegral commented on appendix/device-checklist.md:
```diff
@@ -10,33 +10,35 @@ Before starting, gather a list of all devices to audit that are used for Lullabo
-- [ ] Confirm 1Password is installed with browser extensions.
-- [ ] Confirm an adblocker (uBlock origin, AdGuard) is installed in the default browser.
+- [ ] Confirm all mobile devices have a [screen timeout set to no more than 10 minutes](https://security.lullabot.com/physical/locking.html).
+- [ ] Confirm "Find My" or equivalent is enabled.
```
```suggestion
- [ ] Confirm "Find My" or equivalent is enabled if the team member wants it on.
```

deviantintegral commented on appendix/device-checklist.md:
```diff
@@ -10,33 +10,35 @@ Before starting, gather a list of all devices to audit that are used for Lullabo
+- [ ] Confirm all mobile devices have a [screen timeout set to no more than 10 minutes](https://security.lullabot.com/physical/locking.html).
+- [ ] Confirm "Find My" or equivalent is enabled.
+- [ ] All operating systems should be up to date.
+- [ ] All browsers should be up to date.
```
I don't think this is in our list, but I'm not against adding it. And I'm not sure "all" makes sense, especially where folks may have little used browsers that are behind (until they open it for this check!).

YesCT replied:
Yeah, browsers up to date was on this list, but not our spreadsheet.
I like it on the self checklist.

I agree, I'm unlikely to normally update a browser until I use it.

deviantintegral commented on appendix/device-checklist.md:
```diff
@@ -10,33 +10,35 @@ Before starting, gather a list of all devices to audit that are used for Lullabo
 
-- [ ] Confirm "Find My" or equivalent is enabled.
-- [ ] Confirm firewall is enabled.
+- [ ] All computers have a [complex password](https://security.lullabot.com/access/passwords.html), at least 16 characters long. If you add your passwords to 1password, 1password will tell you the strenght of the password; try to get "excellent" strength. Do NOT use guessable patterns like: family names, birthdays, the word “lullabot”, etc.
```
`1Password` with a capital p, everywhere!

`strenght` - typo

YesCT replied:
Fixed strength typo and 1Password casing. 👍

deviantintegral commented on appendix/device-checklist.md:
```diff
@@ -10,33 +10,35 @@ Before starting, gather a list of all devices to audit that are used for Lullabo
-- [ ] Confirm "Find My" or equivalent is enabled.
-- [ ] Confirm firewall is enabled.
+- [ ] All computers have a [complex password](https://security.lullabot.com/access/passwords.html), at least 16 characters long. If you add your passwords to 1password, 1password will tell you the strenght of the password; try to get "excellent" strength. Do NOT use guessable patterns like: family names, birthdays, the word “lullabot”, etc.
+- [ ] All computer [hard drives are encrypted](https://security.lullabot.com/physical/hard-drive-encryption.html) with FileVault, Bitlocker, or equivalent.
```
```suggestion
- [ ] All computer [hard drives are encrypted](https://security.lullabot.com/physical/hard-drive-encryption.html) with FileVault, BitLocker, or equivalent.
```

YesCT replied:
Change made. Resolving.

deviantintegral commented on appendix/device-checklist.md:
```diff
@@ -10,33 +10,35 @@ Before starting, gather a list of all devices to audit that are used for Lullabo
-- [ ] Confirm firewall is enabled.
+- [ ] All computers have a [complex password](https://security.lullabot.com/access/passwords.html), at least 16 characters long. If you add your passwords to 1password, 1password will tell you the strenght of the password; try to get "excellent" strength. Do NOT use guessable patterns like: family names, birthdays, the word “lullabot”, etc.
+- [ ] All computer [hard drives are encrypted](https://security.lullabot.com/physical/hard-drive-encryption.html) with FileVault, Bitlocker, or equivalent.
+- [ ] Confirm firewall is enabled. Document work tasks that require disabling firewall; share with security@lullabot.com.
```
What tasks are these? Normally you can allow apps through the firewall without disabling it entirely.

YesCT replied:
Asking people to tell us... gives us a chance to help people not disable the firewall. I'll look and see if I had any notes about task specifics from the last checklist round we did. Some notes mention xdebug, or having a household system when at home, and vpn when away. 

But maybe instead this could be firmer wording.

deviantintegral commented on appendix/device-checklist.md:
```diff
@@ -10,33 +10,35 @@ Before starting, gather a list of all devices to audit that are used for Lullabo
+- [ ] Confirm firewall is enabled. Document work tasks that require disabling firewall; share with security@lullabot.com.
 - [ ] Confirm no guest accounts are enabled.
-- [ ] Confirm that if remote access is enabled (SSH, file sharing) that the implications are understood.
+- [ ] Confirm no remote access is enabled (SSH, file sharing). Document work tasks that require disabling firewall; share with security@lullabot.com.
```
Mispaste here?

YesCT replied:
yeah, I meant to customize it to remote access. Updating.

deviantintegral commented on appendix/device-checklist.md:
```diff
@@ -10,33 +10,35 @@ Before starting, gather a list of all devices to audit that are used for Lullabo
-- [ ] All backups are encrypted. Cloud services use a private key that only you know.
+- [ ] If computers have a [backup system](https://security.lullabot.com/physical/backups.html) in place, backups should be encrypted.
+- [ ] Optional, but nice to have: backups support versioning to protection against ransomware style scams.
+- [ ] If using cloud services, use a private key that only you know. (Store it in 1password.)
```
```suggestion
- [ ] If using cloud services for backups, use a private key that only you know. (Store it in 1password.)
```

YesCT replied:
Committed suggestion (manually, the button was not available and the hunk said "outdated".)

YesCT commented:
I think I addressed all the feedback. I also read through it again and removed duplicative "Confirm"s and some other small changes.

@deviantintegral (or others), this is ready for another review.

YesCT commented:
I approved the visual diffs, they caught the intended change here. 

<img width="1475" alt="Screenshot 2025-04-01 at 10 07 54 AM" src="https://github.com/user-attachments/assets/97127e41-7cf5-4552-8c52-bbeae724bae7" />

Using bypass to override the approval needed, Brian approves.
